utils/serde/src/main/java/io/apicurio/registry/utils/serde/strategy/GlobalIdStrategy.java

```diff
@@ -47,14 +47,15 @@
     /**
      * Create InputStream from schema.
      * By default we just take string bytes.
-     *
+     * 
      * @param schema the schema
      * @return schema's input stream
      */
     default InputStream toStream(T schema) {
```

Comment:
We said we're gonna add another strategy for this - ToStreamStrategy ...

Reply:
Yeah it turned out I didn't need it, so left it alone for now.  Since for JSON Schema we never have the schema, `toStream()` will never be called (or at least never be called with anything other than `null`).

Reply:
OK, makes sense